main/lsp/LSPIndexer.h

```diff
@@ -61,11 +66,8 @@ class LSPIndexer final {
     /**
      * Commits the given edit to `initialGS`, and returns a canonical LSPFileUpdates object containing indexed trees
      * and file hashes. Also handles canceling the running slow path.
-     *
-     * If `newHashesOrEmpty` contains file hashes, this function avoids re-computing those hashes.
      */
-    LSPFileUpdates commitEdit(std::unique_ptr<Timer> &latencyTimer, SorbetWorkspaceEditParams &edit,
-                              std::vector<core::FileHash> newHashesOrEmpty);
```

Comment:
Hashes are now contained within the `File` objects of `SorbetWorkspaceEditParams` 🎉